lib/base_service.ts

```diff
@@ -294,11 +294,11 @@ export class BaseService {
       _options = extend(_options, options);
       return _options;
     }
-    // Get credentials from environment properties or Bluemix,
+    // Get credentials from environment properties or IBM Cloud,
     // but prefer credentials provided programatically
     _options = extend(
       {},
-      this.getCredentialsFromBluemix(this.name),
+      this.getCredentialsFromIbmCloud(this.name),
```

Comment:
`IBM` not `Ibm`. I would call it `FromCloud`

Reply:
do we need some sort of approval for this nomenclature?